js/src/modals/ExportAccount/exportAccount.js

```diff
@@ -0,0 +1,201 @@
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import { observer } from 'mobx-react';
+import React, { Component, PropTypes } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { newError } from '~/redux/actions';
+import { personalAccountsInfo } from '~/redux/providers/personalActions';
+import { AccountCard, Button, Portal, SelectionList } from '~/ui';
+import { Input } from '~/ui/Form';
+import { CancelIcon, CheckIcon } from '~/ui/Icons';
+import ExportStore from './exportStore';
+
+import styles from './exportAccount.css';
+
+@observer
+class ExportAccount extends Component {
+  static contextTypes = {
+    api: PropTypes.object.isRequired
+  };
+
+  static propTypes = {
+    accounts: PropTypes.object.isRequired,
+    balances: PropTypes.object.isRequired,
+    newError: PropTypes.func.isRequired,
+    personalAccountsInfo: PropTypes.func.isRequired,
+    onClose: PropTypes.func.isRequired
+  };
+
+  componentWillMount () {
+    const { accounts, newError } = this.props;
+
+    this.exportStore = new ExportStore(this.context.api, accounts, newError, null);
+  }
+
+  render () {
+    const { canExport } = this.exportStore;
+
+    return (
+      <Portal
+        buttons={ [
+          <Button
+            icon={ <CancelIcon /> }
+            key='cancel'
+            label={
+              <FormattedMessage
+                id='export.accounts.button.cancel'
+                defaultMessage='Cancel'
+              />
+            }
+            onClick={ this.onClose }
+          />,
+          <Button
+            disabled={ !canExport }
+            icon={ <CheckIcon /> }
+            key='execute'
+            label={
+              <FormattedMessage
+                id='export.accounts.button.export'
+                defaultMessage='Export'
+              />
+            }
+            onClick={ this.onExport }
+          />
+        ] }
+        onClose={ this.onClose }
+        open
+        title={
+          <FormattedMessage
+            id='export.accounts.title'
+            defaultMessage='Export an Account'
+          />
+        }
+      >
+        { this.renderList() }
+      </Portal>
+    );
+  }
+
+  renderList () {
+    let { accounts } = this.props;
+
+    accounts = Object
+      .keys(accounts)
+      .map((address) => accounts[address]);
```

Comment:
LOL, at first glance it looks confusing.

SelectionList requires an array of objects, rather than an object holding objects.

before:
```
{
  'A': {}, 'B': {}, 'C': {} 
}
```

after:
```
[
  0: {}, 1: {}, 2: {} 
]
```